### PR TITLE
Make CI GREEN

### PR DESCRIPTION
**Pull request by SharpEdgeMarshall**
## Description

Reopen #5599 

The target of this PR is to make CI green so we are not cumulating more errors merging broken things

### CODE REVIEW

File: requirements/default.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 pytz>dev
 billiard>=3.6.0,<4.0
-kombu>=4.6.1,<5.0
```
Comment (auvipy) [inline requirements/default.txt]: revert this.

File: .travis.yml [hunk 2]
```diff
@@ -10,9 +10,9 @@ python:
 os:
     - linux
 stages:
+  - lint
```
Comment (auvipy) [inline .travis.yml]: and this.

File: requirements/default.txt (hunk 1 above)
Comment (SharpEdgeMarshall) [reply requirements/default.txt]: this is what is breaking tests

File: .travis.yml (hunk 2 above)
Comment (SharpEdgeMarshall) [reply .travis.yml]: Why do the linting at the end after all tests?
Usually, linting is done before tests

Comment (auvipy) [reply .travis.yml]: nope

Comment (auvipy) [reply .travis.yml]: we prefer unit first to give a faster review.

Comment (SharpEdgeMarshall) [reply .travis.yml]: ok

**Comment by SharpEdgeMarshall** [comment]
I'm fixing flake8

File: requirements/default.txt (hunk 1 above)
Comment (auvipy) [reply requirements/default.txt]: well, let I am going to release kombu 4.6.2 than

Comment (SharpEdgeMarshall) [reply requirements/default.txt]: when?

**Comment by SharpEdgeMarshall** [comment]
@auvipy Travis is green now, what about appveyor?

File: requirements/default.txt (hunk 1 above)
Comment (SharpEdgeMarshall) [reply requirements/default.txt]: Kombu updated

**Comment by SharpEdgeMarshall** [comment]
@auvipy You didn't take my `assert_ping` function in your PR so now I have to remove it

File: t/integration/test_canvas.py [hunk 3]
```diff
@@ -312,7 +316,13 @@ def assert_ids(r, expected_value, expected_root_id, expected_parent_id):
     assert parent_id == expected_parent_id
 
 
+def assert_ping(manager):
```
Comment (SharpEdgeMarshall) [inline t/integration/test_canvas.py]: This is useless now

Comment (auvipy) [reply t/integration/test_canvas.py]: yes, if you want i can remove it

Comment (SharpEdgeMarshall) [reply t/integration/test_canvas.py]: Why you don't want to keep it?

Comment (auvipy) [reply t/integration/test_canvas.py]: ok keeping. code reuse

**Pull request merged**